test/Component/Builder/ReflectionTest.php

```diff
@@ -0,0 +1,30 @@
+<?php declare(strict_types = 1);
+
+namespace RstGroup\ObjectBuilder\Test\Component\Builder;
+
+use RstGroup\ObjectBuilder\Builder\ParameterNameStrategy\SnakeCase;
+use RstGroup\ObjectBuilder\Builder\Reflection;
+use RstGroup\ObjectBuilder\BuildingError;
+use RstGroup\ObjectBuilder\Test\Object\MixedConstructor;
+
+class ReflectionTest extends BuilderTest
+{
+    public static function setUpBeforeClass(): void
+    {
+        static::$builder = new Reflection(new SnakeCase());
```

Comment:
Why static?

Reply:
Because Builders are stateless and no needs to recreate in every test case.